lib/resolver.coffee

```diff
@@ -41,6 +42,9 @@ yeomanResolver = require('yeoman-environment/lib/resolver')
 exports.getNodeModulesPaths = ->
 	paths = yeomanResolver.getNpmPaths()
 
+	if os.platform() isnt 'win32'
+		paths.unshift('/usr/local/lib/node_modules')
```

Comment:
@Page- What is this `globalPaths` property? Never heard of it and can't find any documentation neither.

Based on your suggestion I found https://www.npmjs.com/package/global-paths, which seems to cover all edge cases I'm already implementing, apart from the ones hardcoded by yeoman already. 

I'll send a PR soon making use of such module.